vahnatai/stack-2-game
# Add AI Players

Adds ability to play against artificial intelligence instead of other users. Games vs AI can also be reloaded from database after server restart.

In-game, the AI just performs random legal moves on its turn.

**Testing:**
- `npm i` to install dependencies
- `npm run db-restart` to refresh DB schema
- `npm start` to run servers
    - join waiting room as a named player
    - choose to play against the AI by clicking the related button
    - play until almost complete
    - restart the server
    - should be able to rejoin the game and finish it
